src/client/components/Table/Table.tsx

```diff
@@ -35,14 +47,11 @@ export function RankingTable({ data, loading }) {
                     </Table.Td>
                     <Table.Td
                         className={cx('', {
-                            [classes.zerg]: race == 'ZERG',
-                            [classes.terran]: race == 'TERRAN',
-                            [classes.protoss]: race == 'PROTOSS',
-                            [classes.random]: race == 'RANDOM',
+                            [raceAssets[race]?.className]: raceAssets[race],
```

Comment:
Just a note: `[raceAssets[race]?.className]: raceAssets[race]` should be `[raceAssets[race]?.className]: true` for simplicity

Reply:
Done